proposals/CHAOSS-Improving-Open-Source-Sustainability.md  

```diff
@@ -0,0 +1,124 @@
+
+More subtle effects are also valuable. For example, clearer understanding of dependencies and patterns of use in software make local ecosystems more able to compose software from smaller packages, each more specialized. Smaller, more specialized packages increase “option value” or the innovative potential from recombination. An increased option value can help fix problems or provide new capabilities in fewer locations, meaning fewer partial, redundant, and desynchronized changes.
+
+The first part in this project will scale the dependency analysis and machine learning components of the CHAOSS/Augur software project. Specifically, we will engage the CHAOSS community in an iterative process of identifying critical sub-ecosystems in package managed libraries, core corporatized infrastructure, and scientific software. We will then scale CHAOSS/Augur’s existing machine learning algorithms to discover, assess, and identify ecosystem-level characteristics that generate an ecosystem map. The map will substantially accelerate the open-source community writ large’s ability to discover opportunities for improvement, and models of practice that are effective in similar ecosystems and sub-ecosystems. 
+
+The second part in this project will identify specific metrics and metrics models necessary to identify critical health concerns within a mapped ecosystem.  Scaling the application of CHAOSS/Augur’s machine learning capabilities will reveal characteristics and similarities within and across projects within an ecosystem. Specifically, Augur takes a collection of projects defined by the user, and conducts local and global clustering analysis, topic modeling, discourse analysis (parts of speech in issues, pull requests, and associated comments), message novelty, message sentiment, probabilistic evaluation of a pull request’s likelihood of acceptance, and anomaly detection (i.e., acceleration and deceleration of distinct types of project and ecosystem activity against a normed model built from a configurable training period). 
+
+**Project 2: Centering DEI.** A key component of CHAOSS is the development of DEI metrics to help, not only ourselves, but others best center DEI within their projects. Improving open source work and the DEI share a common geography. For open source community health, the key for aligning our efforts lies in the application of research and practice illustrating how advancing DEI within open source communities enables us to overcome key open source challenges from new perspectives. More productive teams and healthier communities will emerge through this recognition, the introduction of new points of view, and a growing understanding that open source software faces shared challenges advanced through new alignments.
```

Comment:
How does this work intersect with / follow with CHAOSS's DEI audit in 2021?

Reply:
Hi @epicfaace. The DEI audit was about identifying key areas of DEI work that could be done in CHAOSS specifically. We are currently implementing those. For example, we are implementing processes by which metrics translations are done for our metrics releases (https://github.com/chaoss/translations). The proposal here is about helping to bring those recommendations forward to other open source communities. We would like to make what we learned available to help others.